WindowsServerDocs/identity/ad-ds/deploy/Upgrade-Domain-Controllers-to-Windows-Server-2012-R2-and-Windows-Server-2012.md

```diff
@@ -276,18 +276,18 @@ The following table covers common Active Directory-integrated Microsoft applicat
 
 |Product|Notes|  
 |-----------|---------|  
-|[Microsoft Configuration Manager 2007](http://blogs.technet.com/b/configmgrteam/archive/2012/09/10/support-questions-about-windows-8-and-windows-server-2012.aspx)|Configuration Manager 2007 with SP2 (includes Configuration Manager 2007 R2 and Configuration Manager 2007 R3):<br /><br />-   Windows 8 Pro<br />-   Windows 8 Enterprise<br />-   Windows Server 2012 Standard<br />-   Windows Server 2012 Datacenter **Note:** Though these will be fully supported as clients, there is no plan to add support for deploying these as operating systems by using the Configuration Manager 2007 operating system deployment feature. Also, no site servers or site systems will be supported on any SKU of Windows Server 2012.|  
+|[Microsoft Endpoint Configuration Manager](http://blogs.technet.com/b/configmgrteam/archive/2012/09/10/support-questions-about-windows-8-and-windows-server-2012.aspx)|Configuration Manager 2007 with SP2 (includes Configuration Manager 2007 R2 and Configuration Manager 2007 R3):<br /><br />-   Windows 8 Pro<br />-   Windows 8 Enterprise<br />-   Windows Server 2012 Standard<br />-   Windows Server 2012 Datacenter **Note:** Though these will be fully supported as clients, there is no plan to add support for deploying these as operating systems by using the Configuration Manager 2007 operating system deployment feature. Also, no site servers or site systems will be supported on any SKU of Windows Server 2012.|  
```

Comment:
This article is a tough one, might need input from author @MicrosoftGuyJFlo. ConfigMgr 2007 is no longer supported, so should really remove it altogether. The ConfigMgr 2012 row is technically valid, since that product is still supported. Then we should really insert another row for "Microsoft Endpoint Configuration Manager (current branch)" - and then add content in the notes accordingly. Or just point to our existing [supported configurations](https://docs.microsoft.com/en-us/configmgr/core/plan-design/configs/supported-configurations) that list out the clients and server versions that we support.

So will let you two decide how best to handle this one. 😄

Reply:
@MicrosoftGuyJFlo Do you have any suggested changes based on Aaron's comment? I'm happy to make the edits. Thanks!

Reply:
@aczechowski @MicrosoftGuyJFlo I made the changes suggested by Aaron so it would be easier to visualize. Let me know if that is acceptable or how else it should be changed. Thanks.